test/base.py

```diff
@@ -0,0 +1,46 @@
+import os
+import shutil
+import tempfile
+from pathlib import Path
+from unittest import TestCase
+
+
+class BaseTestCase(TestCase):
+    """Base class for our tests to house shared logic."""
+    _using_fake_hook_tools = False
+    _tmpdir = None
+
+    @property
+    def tmpdir(self):
+        if not self._tmpdir:
+            self._tmpdir = Path(tempfile.mkdtemp())
```

Comment:
It's nice to have a suffix so we can tell what the directory is when we find it lost.